### ISSUE TITLE
Alt Text

### ISSUE THREAD

**Issue by farishkash** [scampbell1/f1-3-c2p1-colmar-academy#2]
Should be using alt text on images, screen readers really rely on this.  Also for further accessibility you can look into ARIA roles and landmarks.
